pkgs/by-name/im/imposm/package.nix

```
@@ -0,0 +1,32 @@
+{ lib, buildGoModule, fetchFromGitHub, leveldb, geos }:
+
+buildGoModule rec {
+  pname = "imposm";
+  version = "0.12.0";
+
+  src = fetchFromGitHub {
+    owner = "omniscale";
+    repo = "imposm3";
+    rev = "v${version}";
+    hash = "sha256-xX4cV/iU7u/g9n7dtkkkCtNOPZK5oyprNHGDUuW+ees=";
+  };
+
+  vendorHash = null;
+
+  buildInputs = [ leveldb geos ];
+
+  ldflags = [
+    "-s -w"
+    "-X github.com/omniscale/imposm3.Version=${version}"
+  ];
+
+  # requires network access
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Imposm imports OpenStreetMap data into PostGIS";
+    homepage = "https://imposm.org/";
+    license = licenses.apsl20;
+    maintainers = with maintainers; [ sikmir ];
+  };
```

```suggestion
    mainProgram = "imposm";
  };
```

> `meta.mainProgram` must be set to the name of the executable which facilitates the primary function or purpose of the package, if there is such an executable in $bin/bin/ (or $out/bin/, if there is no "bin" output).

https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes

It should be set `meta.mainProgram`.